Modules/_testlimitedcapi/long.c

```diff
@@ -746,6 +746,17 @@ pylong_asvoidptr(PyObject *module, PyObject *arg)
     return Py_NewRef((PyObject *)value);
 }
 
+static PyObject *
+pylong_aspid(PyObject *module, PyObject *arg)
+{
+    NULLABLE(arg);
+    pid_t value = PyLong_AsPid(arg);
```

Comment:
This line is built with `#define Py_LIMITED_API 0x030d0000` (defined at the top of long.c). So the limited C API 3.12 implementation is not tested.

I propose to merge the PR like that but consider later to find a way to test multiple versions of the limited C API. IMO we need at least separated .c files in `Modules/_testlimitedcapi/`. For example, ``long_36.c`` for limited C API 3.6?

Reply:
I think that it would be better to define `Py_LIMITED_API` *before* includes, so the same source can be used to generate code for different modules: `_testcapi`, `_testcapi35`, `_testcapi313`, etc. If `PyLong_AsInt` is only defined for the unlimited C API and the limited C API 3.13, its wrapper should be surrounded by `#if`/`#endif`.